src/BackupDestination/BackupDestination.php

```diff
@@ -113,6 +115,17 @@ public function connectionError(): Exception
         return $this->connectionError;
     }
 
+    public function extraOptions(): ?array
+    {
+        $extraConfig = config('filesystems.disks.'.$this->diskName().'.dump_extra_options') ?? [];
```

Comment:
As this does not concern the database dump this option should be renamed. What about `backup_extra_options`?

Reply:
About rename - no problem.  

As for the tests, I'll think about which ones are better to add. Maybe you will have ideas?
Since the config is taken from filesystems,  I think should be need to add a factory and refactor some code more globally.

I will write some tests in the coming days.